Accessibility/audit_procedure.md

```diff
@@ -0,0 +1,13 @@
+### Auditing a page or site for accessibility
+
+1. Define the scope of the audit.
+2. On Google Drive, [create a copy of the Accessibility audit detailed report template](https://docs.google.com/spreadsheets/d/1xTNkQI-rHp0LzyjtBDRGtHFNdyG454BihGmjl71N2bA/copy).
```

Comment:
When I tried to go to this link, I got an "access denied" message for mk8066@princeton.edu (the address associated with my Princeton google account) - might need to check permissions.

Reply:
Good catch!  Fixed.